.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
-    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
what was the reason to change the style from the Project specific for `Default` one? we have `Project.xml` for project level styles.

Reply:
Reverted this Back.